rc/lsp.kak

```diff
@@ -1184,17 +1184,17 @@ $([ -z ${kak_hook_param+x} ] || echo hook = true)
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
-define-command lsp-formatting -docstring "Format document" %{
-    lsp-formatting-request false
+define-command lsp-formatting -params ..1 -docstring "Format document" %{
+    lsp-formatting-request false %arg{1}
```

Comment:
Since this is public we should document the parameter in the docstring. Same below.
(Also in future we can equip this function with a `shell-script-candidates` completer that shows provides server names).